Common.cs

```diff
@@ -237,5 +238,19 @@ static void chk_CheckStateChanged(object sender, EventArgs e)
         {
             Settings.Instance[(string)((CheckBox)(sender)).Tag] = ((CheckBox)(sender)).Checked.ToString();
         }
+
+        public static string getMissingParamsMessage(int missed, System.Collections.Generic.List<string> missing, int maxItems = 25)
+        {
+            string list = "Missing " + missed + " params\n";
+
+            foreach (var item in missing.Take(maxItems))
+            {
+                list += item + "\n";
+            }
+            if (missing.Count > 25)
+                list += "(and " + (missing.Count - 25) + " more).";

```

Comment:
and here

Reply:
Yeah right. I added that param very last minute - it should be maxItems

Reply:
See https://github.com/ArduPilot/MissionPlanner/issues/2519